src/handlers/v1/markets/[chainId]/[marketId]/collateral-swap/index.ts

```diff
@@ -16,10 +16,10 @@ import { validateMarket } from 'src/validations';
 
 type GetCollateralSwapQuotationRouteParams = EventPathParameters<{ chainId: string; marketId: string }> &
   EventBody<{
-    account?: string;
-    withdrawalToken?: common.TokenObject;
-    amount?: string;
-    targetToken?: common.TokenObject;
+    account: string;
```

Comment:
@chouandy, I recall our earlier discussion about typically using `?` for all parameters in EventBody. Could you remind me of the rationale behind that choice?

Reply:
Because the body is provided by the client, it's not something the API can entirely control.
From the API's perspective, it's not guaranteed to have a value, but needs validation to confirm its presence.
Therefore, all parameters on the API side will be considered optional.
If they are essential, validation will be added accordingly.

Reply:
I see. Thanks for the explanation. The change will be removed.

Reply:
There are also several similar modifications in other places, please check as well.